app/javascript/application.js

```diff
@@ -1,3 +1,3 @@
 // Entry point for the build script in your package.json
 import "@hotwired/turbo-rails"
-import "./controllers"
+import "./controllers"
```

Comment:
- [ ] Please follow the project requirement to set up the React components along with the redux store and its reducer. As per [these](https://github.com/microverseinc/curriculum-rails/blob/main/connect-frontend-frameworks/rails_react_webpack.md#:~:text=Create%20the%20App,your%20Greeting%20component.) requirement.
![image](https://user-images.githubusercontent.com/62290545/200145972-ff32a9b5-5b11-4644-9bcd-8f9990987255.png)

Reply:
@ShadyShawkat thanks I will work on that as suggested in the project requirement